src/App.js

```diff
@@ -38,14 +41,15 @@ function App() {
   return (
     <div className="App">
       <img src="https://facebookbrand.com/wp-content/uploads/2018/09/Header-e1538151782912.png?w=100&h=100"/>
-      <h1>hello</h1>
+      <h1>Hello</h1>
       <h2>Welcome {username}</h2>
```

Comment:
if possible try to use first letter capital for username i.e it should be Welcome **R**amesh